lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -1,5 +1,133 @@
 module Cerner
   module Resources
+    R4_APPOINTMENT_ENTRY ||= {
```

Comment:
Appointment just went into Sandbox today. Can you guarantee me that this response matches the one you can get back from the request?

Reply:
The structure will be the same, do we usually keep the exact same response here as a sample?

Reply:
Yeah

Reply:
d7f9c2a

Reply:
+1